uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/implementation/addon/MqttMessageEvent.java

```diff
@@ -45,6 +45,10 @@ public String getConnectionName() {
 
     @Override
     public boolean isMatched(@NonNull EventFilter filter) {
+        return isMatchedPart1(filter) && isMatchedPart2(filter);
```

Comment:
why are there 2 parts?

Reply:
I do not see that they are being called individually

Reply:
Ignore it, that isn't useful in this instance.

Reply:
If you're going to just call method 1 and then method 2, I see no reason why you should split it. Keep the logic together.

Reply:
fixed in a7c07e944243724a0ffa89ecc8db1bf64c21e01d